WeScan/Edit/EditImageViewController.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  EditImageViewController.swift
+//  WeScan
+//
+//  Created by Chawatvish Worrapoj on 7/1/2563 BE.
+//  Copyright © 2563 WeTransfer. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+public protocol EditImageViewDelegate: class {
+    func cropped(image: UIImage?)
```

Comment:
Can you add some documentation to this method to what it does? For example, I can't understand from this method definition why image is an optional.

Reply:
Related to bottom comment. :)